src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed === true,
+  })}
+  >
+    <h2 className="TodoInfo__title">
+      {todo.title}
+    </h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
user is possible null - add check if user exists
```suggestion
    {user && <UserInfo user={todo.user} />}
```